common.js

```diff
@@ -163,6 +163,8 @@ const GitHubHostedPlatforms = [
   'ubuntu-22.04-x64',
   'macos-11-x64',
   'macos-12-x64',
+  'macos-13-x64',
+  'macos-13-arm64',
```

Comment:
This is not yet available AFAIK

Reply:
Per [the recent announcement](https://github.blog/changelog/2023-04-24-github-actions-macos-13-is-now-available/) macOS 13 is indeed available now:

> The macOS 13 (Ventura) beta runner image is now available for GitHub-hosted macOS runners

Reply:
But I think not on aarch64, you can see https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources doesn't list `macos-13-arm64`.
Or can you show me a workflow that runs with `macos-13` on arm64?

Reply:
Ah! Understood.  I know `macos-13-x64` based on a [recent build](https://github.com/dfed/swift-async-queue/actions/runs/4789595543/jobs/8517653310?pr=16#step:2:6). I believed that `arm64` exists due to [this line](https://github.com/actions/runner-images/blob/cc369a2b40e144281096e6fc373c46da5a94b783/images.CI/macos/anka/CreateCleanAnkaTemplate.ps1#L152) but I'm guessing since the docs aren't updated yet. Let me remove these lines for now – they're easy to add in later.

Reply:
```suggestion
```